addons/medical_status/functions/fnc_getCardiacOutput.sqf

```diff
@@ -20,13 +20,18 @@
     Source: http://en.wikipedia.org/wiki/Cardiac_output
 */
 
-// to limit the amount of complex calculations necessary, we take a set modifier to calculate Stroke Volume.
-#define MODIFIER_CARDIAC_OUTPUT     0.1904761
+// Value taken from https://doi.org/10.1093%2Feurheartj%2Fehl336
+// as 94/95 ml ± 15 ml
+#define VENTRICLE_STROKE_VOL 95e-3
 
 params ["_unit"];
 
-private _bloodVolumeRatio = (GET_BLOOD_VOLUME(_unit) / DEFAULT_BLOOD_VOLUME);
+private _bloodVolumeRatio = GET_BLOOD_VOLUME(_unit) / DEFAULT_BLOOD_VOLUME;
 private _heartRate = GET_HEART_RATE(_unit);
-private _cardiacOutput = ((_bloodVolumeRatio / MODIFIER_CARDIAC_OUTPUT) + ((_heartRate / DEFAULT_HEART_RATE) - 1)) / 60;
 
-(0 max _cardiacOutput)
+// Blood volume ratio dictates how much is entering the ventricle (this is an approximation)
+private _entering = linearConversion [0.5, 1, _bloodVolumeRatio, 0, 1, true];
```

Comment:
I'm worried this will block CO and bleeding when blood gets too low (e.g. CPR)

Reply:
I set the conversion based on the bleedout condition at 50% volume so it should never become 0 currently. We will presumably need to change it when dealing with the vitals loop during cardiac arrest stuff (if we remove that death condition).